syntax = "proto3";
package mars.stn;


option java_package = "win.liyufan.im.proto";
option java_outer_classname = "PullGroupMemberResultOuterClass";

//option objc_class_prefix = "LYF";
import "group.proto";

message PullGroupMemberResult {
repeated GroupMember member = 1;
}
